controllers/home.js

```diff
@@ -4,11 +4,11 @@
  */
 
 exports.index = function(req, res) {
-  if (req.user) {
-    res.redirect('/learn-to-code')
-  } else {
+if (reg.user && process.env.NODE_ENV !== 'production'){ //added !== dev because crash
+	res.redirect('/learn-to-code')
```

Comment:
I am seeing 'Failed to lookup view "resources/learn-to-code"' is this what you are seeing?